Revert "Solves issue, enums without own items, just generalizations"

XingZhao-CATR opened the pull request:
Reverts OpenNetworkingFoundation/EAGLE-Open-Model-Profile-and-Tools#202

Problem occured when running the #202  version.
Need to check where is wrong.

XingZhao-CATR committed:
Revert "Solves issue, enums without own items, just generalizations"

demx8as6 commented:
Interesting that it causes issues on your side, while it has solved problems for wireless yang generation ;)
Due to the fact that I have a modified version of the tool, addressing issues only visible in the wireless project, I will merge this pull request. No need that the offical project gets blocked by this.

XingZhao-CATR commented:
Hello Martin

It's good to merge a modified version into the official one no matter what the modification is aimed for. This project should serve for all related researching filed.
I revert your pull request because it seems has some problem when running on my PC. But I still hope you can create a new pull request after it's working fine. I can fix the problem on my side either, that's not a big problem, don't have any other thoughts :)

Thanks & Regards
Xing


Best Regards 
-----------------------------------------------------------------
Zhao Xing  赵星

技术与标准研究所 宽带网络研究部
中国信息通信研究院（原工业和信息化部电信研究院）
China Academy of Information and Communication Technology
(China Academy of Telecom. Research), MIIT. P.R. China

TEL: 86-15652962875
E-mail: zhaoxing@ritt.cn, zhaoxing@caict.ac.cn 
 
From: Martin Skorupski
Date: 2017-03-15 18:14
To: OpenNetworkingFoundation/EAGLE-Open-Model-Profile-and-Tools
CC: XingZhao-CATR; Author
Subject: Re: [OpenNetworkingFoundation/EAGLE-Open-Model-Profile-and-Tools] Revert "Solves issue, enums without own items, just generalizations" (#213)
Interesting that it causes issues on your side, while it has solved problems for wireless yang generation ;)
Due to the fact that I have a modified version of the tool, addressing issues only visible in the wireless project, I will merge this pull request. No need that the offical project gets blocked by this.
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.
